src/Item/Item.service.ts

```diff
@@ -433,7 +429,13 @@ export class ItemService {
       eth_address,
     })
 
-    attributes.local_content_hash = contentHash
+    attributes.blockchain_item_id = dbItem ? dbItem.blockchain_item_id : null
+
+    // Compute the content hash of the item to later store it in the DB
+    attributes.local_content_hash =
+      attributes?.blockchain_item_id && dbCollection?.contract_address
```

Comment:
two things

- is `attributes?` necessary? seems to always be defined right?
- shouldn't this be a function with a name? it's also used elsewhere like in `buildStandardWearableEntityMetadata`

Reply:
I didn't understand the question about attributes. I'm using attributes there to have an updated representation of what's going to be inserted into the database to compute its hash.
About the second one, it's similar to the one in `buildStandardWearableEntityMetadata`, as this one checks that the `dbCollection` exists, but I've refactored into a function so we can name it.

Reply:
Oh, you're right! I've changed the code to use the function that checks if the item was published, this code is outdated